### DISCUSSION TITLE
Unable to resolve path for entity

### DISCUSSION THREAD

**Discussion by CECI-LALA** [Open-Systems-Pharmacology/Forum#1803] [Questions & Problems]
Dear community,

I am trying to build a large molecule model. I made some modifications in MOBI and attempted to update it to the simulation, but encountered a path issue and generated an error. 

<img width="1062" alt="problem" src="https://github.com/user-attachments/assets/2db2161c-7963-45e9-99e7-48e2e0e0aa55">

The setting of endogenous IgG cannot reach the plasma and interstitial of organs, however, previous attempts to modify it have not been done in this way. 

Which part should I check to eliminate this path issue? 
The current method I can think of is to manually fill in the corresponding parameters in each organ.

Thank you!

**Comment by bhim1992** [comment]
I also had the same issue with small-molecule
